### PR TITLE
 Added install_github_r_package for installing r packages from remotes

### PR DESCRIPTION
**Pull request by muschellij2**
## Description
Created simple wrapper `install_github_r_package` for installing r packages from `remotes`.  Performs a check, similar to `plumber:::droplet_capture` to see if the package is installed already; if not, it installs `remotes`.  Then uses `remotes::install_github` to install the

## Example
```r
install_github_r_package("sckott/analogsea")
```

Did not find tests for `install_r_package` so wasn't sure where to put `install_github_r_package` tests.

### CODE REVIEW

**Comment by sckott** [comment]
thanks very much @muschellij2 

A few things:

1. There's a bunch of white space changes - try to keep those to a minimum
2. There's two functions added. `droplet_ls` was introduced in the PR but not mentioned. Open a separate PR for that. I'm not sure I'd accept it though since `ls` is pretty simple to just do as `droplet_ssh(droplet, "ls")` 
3. You don't need to namespace fxn calls within the pkg that those fxns are from, e.g, `foo::bar()` can just be `bar()`

it's really hard to test this pkg, but does the `install_github_r_package` fxn work for you? i can test it too.

**Comment by muschellij2** [comment]
Sorry:

1. WS came from RStudio loading and formatting.
2. Removed `droplet_ls`.  Agree it's simple, but I've found myself having to do it 5-6 times, and it seemed nice to have convenience function.
3.  `install_github_r_package` has worked for me.

**Comment by sckott** [comment]
thanks, i'll test it out and get back here

File: R/debian.R [hunk 1]
```diff
@@ -108,6 +108,48 @@ debian_apt_get_install <- function(droplet, ...) {
...
-    sprintf("Rscript -e \"install.packages(\'%s\', repos=\'%s/\')\"", package, repo)
+              sprintf("Rscript -e \"install.packages(\'%s\', repos=\'%s/\')\"", package, repo)
   )
 }
+
+#' @rdname debian
+#' @export
+#' @param package Name of R package to install.
+#' @param repo CRAN mirror to use.
+install_github_r_package <- function(droplet, package, repo = "http://cran.rstudio.com") {
```
Comment (sckott) [inline R/debian.R]: lets change the repo url to `https://cloud.r-project.org/` - and can you do that for the `install_r_package` function as well?

Comment (muschellij2) [reply R/debian.R]: Done

**Comment by sckott** [comment]
thanks

**Pull request merged**